### PR TITLE
Add button class to client fingerprint popover

### PR DESCRIPTION
**Pull request by mylesw27**
We are missing a `.btn` class on the client fingerprint popover button in `instructorAssessmentInstance`. The missing class is resulting in an undesired border on the button as seen below: 
Screenshot 2024-10-24 at 8 47 37 AM
Adding this class will restore the button to the desired badge appearance: 
Screenshot 2024-10-24 at 8 49 07 AM

### CODE REVIEW

File: apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts [hunk 1]
```diff
@@ -582,7 +582,7 @@ export function InstructorAssessmentInstance({
                               <td>
                                 <button
                                   type="button"
-                                  class="badge color-${FINGERPRINT_COLORS[
+                                  class="btn badge color-${FINGERPRINT_COLORS[
```
Comment (mwest1066) [inline apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts:585]: I'm not really sure about combining the `btn` and `badge` styling here. Would it be better to fully style these as buttons (probably `btn-sm` or `btn-xs`)? Or does that interfere with the colors?

Comment (mylesw27) [reply apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts]: That makes sense and should not interfere with the colors. Updated to `btn-xs` in [3f9055a](https://github.com/PrairieLearn/PrairieLearn/pull/10853/commits/3f9055a3beb2f7f1a84941d6e7d020d63011604b).

Comment (jonatanschroeder) [reply apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts]: A quick search brought up this:
https://github.com/PrairieLearn/PrairieLearn/blob/38b83c3a1e5cb95451863ffaa6a531826eae745d/apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts#L679
https://github.com/PrairieLearn/PrairieLearn/blob/38b83c3a1e5cb95451863ffaa6a531826eae745d/apps/prairielearn/elements/pl-integer-input/pl-integer-input.mustache#L76

So it's not unprecedented, but I see your concern. If this is changed to btn, though, we may want to revisit the cases above (though I believe the element case is being handled by #10701).

**Review by jonatanschroeder — APPROVED** [review]

File: apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts (hunk 1 above)
Comment (nwalters512) [reply apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts]: I opened #10859 to fix the styling on the student assessment instance page. The styles in elements will indeed be fixed by the PR that you linked.

**Comment by nwalters512** [comment]
I noticed that these `color-*` classes combined with `btn` result in bad focus styles and thus worse accessibility. I'm going to see if I can improve that a bit in another PR before we land this.

**Comment by jonatanschroeder** [comment]
> I noticed that these `color-*` classes combined with `btn` result in bad focus styles and thus worse accessibility. I'm going to see if I can improve that a bit in another PR before we land this.

Related: #10855.

**Review by mwest1066 — APPROVED** [review]

**Comment by nwalters512** [comment]
@mylesw27 this ended up getting done in #10860 - it was easiest to just audit and update all usages of these custom colors at once! Thanks for taking this initiative to get this ball rolling 🙏 we can close this now.

**Pull request closed without merging**